### ISSUE TITLE
How incompatible is this fork with the original Blitz3D?

### ISSUE THREAD

**Issue by IndependentCommunity** [ZiYueCommentary/Blitz3D#20]
As I understand it, commands related to fonts are not compatible, and there also seems to be incompatibility with the connection of userlib decls?

**Comment by IndependentCommunity** [comment]
This is not a bug. I'm just wondering how incompatible this version is with the original. For example, in the original version this code works, in yours it returns "Too many parameters" error.

```
; Load fonts to a file handle variables
fntArial=LoadFont("Arial",24)
fntArialB=LoadFont("Arial",18,True)
fntArialI=LoadFont("Arial",32,False,True)
fntArialU=LoadFont("Arial",14,False,False,True)
```

**Comment by ZiYueCommentary** [comment]
> This is not a bug. I'm just wondering how incompatible this version is with the original. For example, in the original version this code works, in yours it returns "Too many parameters" error.
> 
> ```
> ; Load fonts to a file handle variables
> fntArial=LoadFont("Arial",24)
> fntArialB=LoadFont("Arial",18,True)
> fntArialI=LoadFont("Arial",32,False,True)
> fntArialU=LoadFont("Arial",14,False,False,True)
> ```

Blitz3D TSS using FreeType for the font system, instead of Windows API in vanilla. FreeType does not support loading font from font name, since it is not particular enough. FreeType is support loading font from the file only. Therefore, `bold`, `underline`, `strikethrough` is removed since they are being used for searching for match font.

If you want to load fonts from the font name, you have to use `FontPath` to find the font's file and pass the path in to `LoadFont`. Note that this function is not available for WINE.

**Comment by ZiYueCommentary** [comment]
Incompatible which related to fonts is fixed in #22
